src/super_gradients/recipes/checkpoint_params/default_checkpoint_params.yaml

```diff
@@ -1,5 +1,6 @@
 load_checkpoint: False # whether to load checkpoint
 load_backbone: False # whether to load only backbone part of checkpoint
+checkpoint_path: # checkpoint path that is located in super_gradients/checkpoints
```

Comment:
Whats this doing here ?

Reply:
It seems that it is now required to have "checkpoint_path" defined (even if it's value is None):
Check https://github.com/Deci-AI/super-gradients/blob/fe011b2172768a17a9795071647c5efe6041da4f/src/super_gradients/training/sg_trainer/sg_trainer.py#L210-L217